module/VuFind/src/VuFind/View/Helper/Root/Config.php

```diff
@@ -88,4 +88,15 @@ public function ajaxCoversEnabled()
     {
         return $this->get('config')->Content->ajaxcovers ?? false;
     }
+
+    /**
+     * Should we limit the number of subjects displayed on the full record?
+     *
+     * @return int or float (INF)
+     */
+    public function subjectLimit()
+    {
+        $subjectLimit = $this->get('config')->Site->subjectLimit;
+        return $subjectLimit ? (int)$subjectLimit : INF;
```

Comment:
The INF constant is neat, but I wonder if PHP_INT_MAX would serve the same basic function while making it easier to have a consistent return type on the function.